circuit_maintenance_parser/parsers/seaborn.py

```diff
@@ -0,0 +1,102 @@
+"""Telstra parser."""
+import logging
+import re
+from typing import Dict, List
+
+from dateutil import parser
+from bs4.element import ResultSet  # type: ignore
+
+from circuit_maintenance_parser.errors import ParserError
+from circuit_maintenance_parser.parser import CircuitImpact, Html, Impact, Status, Subject
+
+# pylint: disable=too-many-branches
+
+
+logger = logging.getLogger(__name__)
+
+
+class SubjectParserSeaborn1(Subject):
+    """PArser for Seaborn subject string."""
+
+    def parse_subject(self, subject, data_base):
+        data = data_base.copy()
+        try:
+            search = re.search(r".+\[## ([0-9]+) ##\].+", subject)
+            if search:
+                data["account"] = search.group(1)
+            return [data]
+
+        except Exception as exc:
+            raise ParserError from exc
+
+
+class HtmlParserSeaborn1(Html):
+    """Notifications Parser for Seaborn notifications."""
+
+    def parse_html(self, soup, data_base):
+        """Execute parsing."""
+        data = data_base.copy()
+        try:
+            self.parse_body(soup, data)
+            return [data]
+
+        except Exception as exc:
+            raise ParserError from exc
+
+    def parse_body(self, body, data):
+        p_elements = body.find_all("p")
+
+        for index, element in enumerate(p_elements):
+            if "NOTIFICATION TYPE" in element.text:
+                pass
+            elif "DESCRIPTION" in element.text:
+                pass
+            elif "SERVICE IMPACT" in element.text:
+                pass
+            elif "LOCATION" in element.text:
+                pass
+            elif "SCHEDULE" in element.text:
+                schedule = p_elements[index + 1].text
+                start, end = schedule.split(" - ")
+                data["start"] = self.dt2ts(parser.parse(start))
+                data["end"] = self.dt2ts(parser.parse(end))
+                data["status"] = Status("CONFIRMED")
+            elif "AFFECTED CIRCUIT" in element.text:
+                circuit_id = element.text.split(": ")[1]
+                data["circuits"] = [(CircuitImpact(impact=Impact("OUTAGE"), circuit_id=circuit_id))]
+
+
+class HtmlParserSeaborn2(Html):
+    """Notifications Parser for Seaborn notifications."""
```

Comment:
following the patter you introduced in the previous parser, an example of the HTML data would be great!

Reply:
Do you think this is a good patterned to follow? It makes the file very large when the user could check the tests for the examples?

Reply:
I believe so. The tests are good as examples, but sometimes they come with some boiler plate. 
Even this looks verbose and makes the  docstring larger, it's much easier to understand it later, specially for someone new to it.

Reply:
Okay sure will add some in.